content/docs/tasks/traffic-management/egress-gateway/index.md

```diff
@@ -0,0 +1,535 @@
+    {{< /text >}}
+
+    Note that we redirected only the traffic from the port 80 to the egress gateway, the HTTPS traffic to the port 443 went directly to _edition.cnn.com_.
+
+### Cleanup of the egress gateway for HTTP traffic
+
+Remove the previous definitions before proceeding to the next step:
+
+{{< text bash >}}
+$ istioctl delete gateway istio-egressgateway
+$ istioctl delete serviceentry cnn
+$ istioctl delete virtualservice direct-through-egress-gateway
+{{< /text >}}
+
+## Perform TLS origination with the egress `Gateway`
+
+Let's perform TLS origination with the egress `Gateway`, similar to the [TLS Origination for Egress Traffic](/docs/tasks/traffic-management/egress-tls-origination/) task.  Note that in this case the TLS origination will be done by the egress Gateway server, as opposed to by the sidecar in the previous task.
+
+1.  Create an egress `Gateway` for _edition.cnn.com_, port 443.
+
+    If you have [mutual TLS Authentication](/docs/tasks/security/mutual-tls/) enabled in Istio:
+
+    {{< text bash >}}
+    $ cat <<EOF | istioctl create -f -
+    kind: Gateway
+    metadata:
+      name: istio-egressgateway
+    spec:
+      selector:
+        istio: egressgateway
+      servers:
+      - port:
+          number: 443
+          name: https
+          protocol: HTTPS
+        hosts:
+        - "*"
+        tls:
+          mode: MUTUAL
+          serverCertificate: /etc/certs/cert-chain.pem
+          privateKey: /etc/certs/key.pem
+          caCertificates: /etc/certs/root-cert.pem
+    EOF
+    {{< /text >}}
+
+    otherwise:
+
+    {{< text bash >}}
+    $ cat <<EOF | istioctl create -f -
+    kind: Gateway
+    metadata:
+      name: istio-egressgateway
+    spec:
+      selector:
+        istio: egressgateway
+      servers:
+      - port:
+          number: 443
+          name: http-port-for-tls-origination
+          protocol: HTTP
+        hosts:
+        - "*"
+    EOF
+    {{< /text >}}
+
+1.  Define a `ServiceEntry` for `edition.cnn.com` and a `VirtualService` to direct
+the traffic through the egress gateway:
+
+    {{< text bash >}}
+    $ cat <<EOF | istioctl create -f -
+    apiVersion: networking.istio.io/v1alpha3
+    kind: ServiceEntry
+    metadata:
+      name: cnn
+    spec:
+      hosts:
+      - edition.cnn.com
+      ports:
+      - number: 80
+        name: http-port
+        protocol: HTTP
+      - number: 443
+        name: http-port-for-tls-origination
+        protocol: HTTP
+      resolution: DNS
+    ---
+    apiVersion: networking.istio.io/v1alpha3
+    kind: VirtualService
+    metadata:
+      name: direct-through-egress-gateway
+    spec:
+      hosts:
+      - edition.cnn.com
+      gateways:
+      - istio-egressgateway
+      - mesh
+      http:
+      - match:
+        - gateways:
+          - mesh
+          port: 80
+        route:
+        - destination:
+            host: istio-egressgateway.istio-system.svc.cluster.local
```

Comment:
what is the use of this route?

Reply:
This route is to direct the traffic thru the egress gateway.